LICENSE-MIT

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2016 rustendo64 Developers
```

Comment:
Same with this, I'd like to retain copyright under `Jake Taylor` instead of `rustendo64 Developers` for the time being. I'll probably switch to that later if we get significant contributions, but for now it simplifies things to keep it under my name.